frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -216,6 +216,44 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
       },
     };
 
+    const rbacName = `${selectedTargetNamespace}-prometheus`;
+    const role = {
+      kind: RoleModel.kind,
+      apiVersion: `${RoleModel.apiGroup}/${RoleModel.apiVersion}`,
+      metadata: {
+        name: rbacName,
+        namespace: selectedTargetNamespace,
+      },
+      rules: [
+        {
+          apiGroups: [''],
+          resources: ['services', 'endpoints', 'pods'],
+          verbs: ['get', 'list', 'watch'],
+        },
+      ],
+    };
+
+    const roleBinding = {
```

Comment:
```suggestion
    const prometheusRoleBinding: K8sResourceKind = {
```

Reply:
wanted to use it but the `roleRef` field is not assignable `K8sResourceKind`:

`Type '{ kind: string; apiVersion: string; metadata: { name: string; namespace: any; }; roleRef: { kind: string; name: string; apiGroup: string; }; subjects: { kind: string; name: string; namespace: string; }[]; }' is not assignable to type 'K8sResourceKind'.
  Object literal may only specify known properties, and 'roleRef' does not exist in type 'K8sResourceKind'.`